src/choose_block_size.jl

```diff
@@ -7,3 +7,21 @@ function choose_block_size(C, A, B, ::Nothing)
 end
 
 choose_block_size(C, A, B, block_size::Integer) = block_size
+
+function choose_parameter(C, A, B, ::Nothing)
+    m = Int128(size(A, 1))
+    n = Int128(size(B, 1))
+    k = Int128(size(A, 2))
```

Comment:
Ha, hopefully we won't be multiplying matrices that are that large.

Reply:
Haha, yeah, I kinda blindly just copied it from

https://github.com/MasonProtter/Gaius.jl/blob/b39d597f8ff3435e92fcb6d9bb3ff7fd2ae850c5/src/choose_block_size.jl#L1-L2

I guess you could construct a couple of terabyte arrays in a really large machine but it's probably impossible to finish O(n^3) computation in a meaningful time using shared memory parallelism... But who knows...